src/microbe_stage/MicrobeAI.cs

```diff
@@ -42,6 +43,13 @@ public class MicrobeAI
     [JsonProperty]
     private float pursuitThreshold;
 
+    /// <summary>
+    ///   A value between 0.0f and 1.0f, this is the portion of the microbe's atp bar that needs to refill
+    ///   before resuming motion.
+    /// </summary>
+    [JsonProperty]
+    private float atpTreshold;
```

Comment:
Isn't this a typo?

Reply:
yes, yes it is. I think I'm now 0 for 3 on PRs in this repo.